test/controllers/CheckYourAppealController.test.ts

```diff
@@ -1,27 +1,25 @@
-import 'reflect-metadata'
-import '../../src/controllers/CheckYourAppealController'
-import { createApplication } from "../ApplicationFactory";
-import { RedisService } from "../../src/services/RedisService";
-import { createSubstituteOf } from "../SubstituteFactory";
-import * as request from "supertest";
-import { SUBMISSION_SUMMARY_PAGE_URI } from "../../src/utils/Paths";
-import { OK } from "http-status-codes";
+import 'reflect-metadata';
+import '../../src/controllers/CheckYourAppealController';
+import * as request from 'supertest';
+import { SUBMISSION_SUMMARY_PAGE_URI } from '../../src/utils/Paths';
+import { OK } from 'http-status-codes';
 import { expect } from 'chai';
+import { createApp, getDefaultConfig } from '../ApplicationFactory';
+import { createFakeSession } from '../utils/session/FakeSessionFactory';
 
-const app = createApplication(container => {
-    container.bind(RedisService).toConstantValue(createSubstituteOf<RedisService>());
-});
+const config = getDefaultConfig();
 
 describe('CheckYourAppealController', () => {
     describe('GET request', () => {
-        it('should return 200 when trying to access the submission summary', async () => {
-
+        it('should return 200 when trying to access the submission summary with a session ', async () => {
```

Comment:
Not sure how much value that tests add. Not a concern of this PR but subject to removal in the future.